.github/workflows/ci.yml

```diff
@@ -289,26 +284,23 @@ jobs:
           username: ansys-bot
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Cache 23.1 API Code
+      - name: Cache API Code
         uses: actions/cache@v4
-        id: cache-231-api-code
+        id: cache-api-code
         with:
           path: src/ansys/fluent/core/generated
-            doc/source/api/core/meshing/tui
-            doc/source/api/core/meshing/datamodel
-            doc/source/api/core/solver/tui
-            doc/source/api/core/solver/datamodel
-          key: API-Code-v${{ env.API_CODE_CACHE }}-${{ steps.version.outputs.PYFLUENT_VERSION }}-v23.1.0-${{ hashFiles('src/ansys/fluent/core/codegen/**') }}
-          restore-keys: API-Code-v${{ env.API_CODE_CACHE }}-${{ steps.version.outputs.PYFLUENT_VERSION }}-v23.1.0
+          # Combined cache key for all versions:
```

Comment:
As restored path is identical for all Fluent version, it makes sense to use a combined cache.